modules/microshift-backing-up-manually.adoc

```diff
@@ -4,25 +4,32 @@
 
 :_content-type: PROCEDURE
 [id="microshift-backing-up-manually_{context}"]
-= Backing up {product-title} data manually
+= Backing up {microshift-short} data manually
 
-You can back up {product-title} data manually at any time. Data backups are advised before any system updates. Backups are created in the `/var/lib/microshift-backups` directory by default. You can also specify a directory by adding custom parameters to the required backup command.
+You can back up {microshift-short} data manually at any time. Data backups are advisable before any system updates. Automated backups are created in the `/var/lib/microshift-backups` directory. You can use this directory for manually backing up and restoring data by specifying it in each command. When you create a backup, you must use the entire file path.
 
 [NOTE]
 ====
-On `rpm-ostree` systems, {product-title} creates an automatic backup on every start and restores data automatically.
+On `rpm-ostree` systems, {microshift-short} creates an automatic backup on every start and restores data automatically.
 ====
 
 .Prerequisites
+
 * You have root access to the host.
-* {product-title} is stopped.
+* {microshift-short} is stopped.
+
+[NOTE]
+====
+You cannot use the `microshift backup` command if the `microshift.service` status is `failed`.
+====
 
 .Procedure
-. Manually create a backup by using the default name and parent directory, `/var/lib/microshift-backups/<default-backup-name>`, by running the following command:
+
+. Manually create a backup by using the parent directory and specifying a name, such as `/var/lib/microshift-backups/<my_manual_backup>`, by running the following command:
 +
 [source,terminal]
 ----
-$ sudo microshift backup
+$ sudo microshift backup /var/lib/microshift-backups/<my_manual_backup>
```

Comment:
```suggestion
$ sudo microshift backup /var/lib/microshift-backups/my_manual_backup
```

We need to be careful about using `<` and `>` in command examples because they change the shell behavior.

Reply:
@dhellmann It is docs style for user-input values. I can add the note about replacing the value. https://docs.engineering.redhat.com/display/CCSENABLE/Command+line